docs/loading/Flink-connector-starrocks.md

```diff
@@ -1,173 +1,647 @@
+| sink.connect.timeout-ms           | No           | 1000              | The timeout for establishing HTTP connection. Valid values: 100 to 60000. Unit: ms.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| sink.wait-for-continue.timeout-ms | No           | 10000             | Supported since 1.2.7. The timeout for waiting response of HTTP 100-continue from FE. Valid values: 3000 to 600000. Unit: ms                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| sink.ignore.update-before         | No           | true              | Supported since version 1.2.8. Whether to ignore `UPDATE_BEFORE` records from Flink when loading data to primary key table. If false, will treat the record as a delete operation to StarRocks table.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/69b927b2-5630-4240-9672-8b233406ee52)
This sentence does not make sense. Change to -> If this parameter is set to `false`, xxx.
And I can't understand what "treat the record as a delete operation to StarRocks table" means.